graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/exception/KotlinDataFetcherExceptionHandler.kt

```diff
@@ -24,19 +24,24 @@ import graphql.execution.DataFetcherExceptionHandlerResult
 import org.slf4j.LoggerFactory
 
 /**
- * Default [DataFetcherExceptionHandler] used by all GraphQL execution strategies. All exceptions thrown during GraphQL execution are logged and then returned as wrapped
- * [ExceptionWhileDataFetching] error.
+ * Default DataFetcherExceptionHandler used by all GraphQL execution strategies. All exceptions thrown during GraphQL execution are logged and then returned as wrapped
```

Comment:
lets keep the links (https://kotlinlang.org/docs/reference/kotlin-doc.html#linking-to-elements)

Reply:
The links actually don't work in dokka. They can't link to dependency jars